src/test/scala/nl.knaw.dans.easy.solr4files/components/DDMSpec.scala

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.io.FileInputStream
+import java.net.{ HttpURLConnection, URL }
+
+import org.scalatest.{ FlatSpec, Matchers }
+
+import scala.util.Try
+import scala.xml.XML
+
+class DDMSpec extends FlatSpec with Matchers {
+
+  "solrLiteral" should "return proper values" in {
+    assume(canConnectToEasySchemas)
+    val ddm = new DDM(resource.managed(new FileInputStream(
+      "src/test/resources/vault/stores/pdbs/bags/9da0541a-d2c8-432e-8129-979a9830b427/metadata/dataset.xml"
```

Comment:
preferably use the copy in `target` for this using `getClass.getResource()`. See for example

https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/69957d9f2244092f88d49eb903c6cb0bc781ee32/src/test/scala/nl.knaw.dans.easy.multideposit/BlackBoxSpec.scala#L62-L63